src/combatsimulator/equipment.js

```diff
@@ -21,9 +21,16 @@ class Equipment {
     getCombatStat(combatStat) {
         let multiplier = enhancementLevelTotalMultiplierTable[this.enhancementLevel];
 
-        let stat =
-            this.gameItem.equipmentDetail.combatStats[combatStat] +
-            multiplier * this.gameItem.equipmentDetail.combatEnhancementBonuses[combatStat];
+        let bonus = multiplier * this.gameItem.equipmentDetail.combatEnhancementBonuses[combatStat];
```

Comment:
Also not really sure if `enhancementLevelTotalMultiplierTable` is up-to-date when looking at the guide percentages